assign-doi-datacite/src/main/java/no/unit/nva/doi/datacite/mdsclient/NoCredentialsForCustomerRuntimeException.java

```diff
@@ -1,6 +1,15 @@
 package no.unit.nva.doi.datacite.mdsclient;
 
+import no.unit.nva.doi.datacite.clients.exception.ClientException;
 import no.unit.nva.doi.datacite.clients.exception.ClientRuntimeException;
 
 public class NoCredentialsForCustomerRuntimeException extends ClientRuntimeException {
+
+    public NoCredentialsForCustomerRuntimeException() {
+        super();
```

Comment:
ditto

Reply:
Isn't the class name descriptive enough?

Reply:
Anyhow, pushed 9c41e1d80cda674a91114269f6bf071fdd3dcb7a which makes this class pass a exception message to `ClientRuntimeException`.